app/javascript/components/public_health/query/AdvancedFilter.js

```diff
@@ -273,6 +276,26 @@ class AdvancedFilter extends React.Component {
     });
   };
 
+  // Reset modal when cancelled
+  cancel = () => {
+    // check if there was a filter applied
+    const appliedFilterId = localStorage.getItem(`SaraFilter`);
+    const appliedFilter = this.state.savedFilters.find(filter => {
```

Comment:
Typical dave nitpick, but this can be a one-liner `this.state.savedFilters.find(filter => filter.id === parseInt(appliedFiltlerId)`

Reply:
done